src/core/api/BbbPluginSdk.ts

```diff
@@ -202,10 +203,14 @@ export abstract class BbbPluginSdk {
           '': () => { },
         },
         getSessionToken: () => getSessionToken(),
+        logger: pluginLogger,
```

Comment:
Since the HTML5 client now creates and assigns its own logger to `window.bbb_plugins[uuid]`, do we still need to keep using the SDK's `pluginLogger`?